doc/releases/release-notes-3.7.rst

```diff
@@ -332,7 +332,7 @@ Boards & SoC Support
 
   * On :ref:`ST STM32H7B3I Discovery Kit <stm32h7b3i_dk_board>`: ``stm32h7b3i_dk_board``,
     enabled full cache management, Chrom-ART, double frame buffer and full refresh for
-    optimal LVGL perfomances.
+    optimal LVGL performances.
```

Comment:
I wonder if this should be in singular rather than plural. At least it'd sound more natural to me if it was just "optimal LVGL performance"

Reply:
OK, Got it.